polkadot/runtime/parachains/src/lib.rs

```diff
@@ -78,6 +78,13 @@ pub trait FeeTracker {
 	fn decrease_fee_factor(id: Self::Id) -> FixedU128;
 }
 
+/// Trait for reporting rewards for processing parachains blocks
+pub trait RewardsReporter<C> {
```

Comment:
we can move this trait to frame-support, and then implement it on individual pallets.